cron/3.queueProcess.nolock.php

```diff
@@ -151,6 +156,7 @@
         addLabel($kill,  $totalValue > 10000000000, '10b+');
         addLabel($kill,  $totalValue > 100000000000, '100b+');
         addLabel($kill,  $totalValue > 1000000000000, '1t+');
+        addLabel($kill,  $kill["isCapital"], 'capitals');
```

Comment:
I'm open to a more obvious name, as "capitals" might confuse users into thinking this label would also show killmails where capitals were only _involved_, rather than as the victim only.